packages/firestore/test/util/api_helpers.ts

```diff
@@ -46,15 +46,16 @@ export const FIRESTORE = new Firestore({
 });
 
 export function firestore(): Firestore {
+  FIRESTORE.settings({timestampsInSnapshots: true});
```

Comment:
Not sure about this change. Can/should I modify `FIRESTORE` right after it is defined in the global scope?

Reply:
I think it would be fine to move this out of the firestore() function, if that's what you're asking.  I don't have a strong preference.

Reply:
Ok, I guess I'll merge as is, then.